lib/jekyll/commands/new.rb

```diff
@@ -121,23 +121,23 @@ def scaffold_path
         # unless the user opts to generate a blank blog or skip 'bundle install'.
 
         def after_install(path, options = {})
-          Jekyll.logger.info "New jekyll site installed in #{path.cyan}."
-          Jekyll.logger.info "Bundle install skipped." if options["skip-bundle"]
-
           unless options["blank"] || options["skip-bundle"]
             bundle_install path
           end
+
+          Jekyll.logger.info "New jekyll site installed in #{path.cyan}."
+          Jekyll.logger.info "Bundle install skipped." if options["skip-bundle"]
         end
```

Comment:
> Doesn't it seem like we should actually install Jekyll before we claim that Jekyll was installed?

*One wouldn't be able to run `jekyll new ` if Jekyll wasn't installed already.*  :grin:   
(I get it that was not your point.. :grinning: )  

By the time `install-success message` is displayed, the new project site would have already been created at the path specified.
The user is then informed of the initiation of the automated `bundle install` .. (which runs in context to the newly generated `Gemfile`)

But like I mentioned in the previous comment, if Parker decides to stick with his vision, he'll have edit the test file accordingly as well.